apple/LayoutReanimation/REASwizzledUIManager.mm

```diff
@@ -28,12 +29,22 @@ - (id)animationsManager
 
 @implementation REASwizzledUIManager
 
+std::atomic<uint> isFlushingBlocks;
```

Comment:
this isn't necessary but a common pattern for implementation-only private variables is to put them in a braces as follows:

```objc
@implementation REASwizzledUIManager {
  std::atomic<uint> isFlushingBlock;
 ....
}
```

Reply:
I intentionally did it this way because when I want to use the private field defined inside `{ ...  }`, it is interpreted as a pointer to private fields in `UIManager` instead of as a field in `SwizzledUIManager`.